### DISCUSSION TITLE
these relative modules were not found when I use @vant/cli

### DISCUSSION THREAD

**Discussion by qieqie7** [pnpm/pnpm#4054] [Q&A]
```
mkdir test

pnpm init vant-cli-app
```

Install with  `yarn`  by default.

I can start the project using `yarn dev `

When I delete node_modules. I use `pnpm install` to install dependencies.

I can't start the project using `pnpm run dev`.

error mssasge:
![image](https://user-images.githubusercontent.com/30360513/144249784-2de537d7-2c1e-4252-ab06-07276ea4da04.png)

thx for help.

**Comment by zkochan** [comment]
I don't get this error. I selected Vue3 and Less.

**Reply by qieqie7** [reply]
vue2 and Less.
